minio/minioadmin.py

```diff
@@ -163,7 +164,7 @@ def _url_open(
             self,
             method: str,
             command: _COMMAND,
-            query_params: DictType | None = None,
+            query_params: Mapping | None = None,
```

Comment:
Do not change `DictType`. It is valid!

Reply:
My IDE says that using mapping is correct :)

Since the type of dict is dynamic

Reply:
https://github.com/minio/minio-py/blob/master/minio/helpers.py#L82 is what we want.

Reply:
Fixed